resources/views/layouts/app.blade.php

```diff
@@ -2,7 +2,7 @@
 $print = app('printservice')->isPrintMode();
 $pClass = 'App\Http\Controllers\GenericPagesController';
 if (env('APP_ENV') != 'testing') {
-    $action = request()->route()->getAction();
+    $action = request()->route()?->getAction();
```

Comment:
This should fix the error we're seeing in Sentry, but it isn't the root of the error.